app/assets/stylesheets/z_custom.scss

```diff
@@ -0,0 +1,302 @@
+}
+
+
+.contact h1, .about h5{
+  float:right;
+}
+
+hr{
+  border: 5px solid #fff;
+}
+
+/* NAV */
+
+
+.top-nav ul li, .main-nav ul li {
+  display:inline-block;
+  float:left;
+}
+
+.top-nav ul li{
+  float:right;
+  padding:0px 10px;
+}
+
+.main-nav{
+  background:url(image_path("nav-ribbon.png")) no-repeat;
+  background-size: 100% 110px;
+  font-size: 30px;
+  min-height: 130px;
+  // margin: -70px auto 0;
+}
+
+.main-nav ul{
+  margin-left:80px;
+}
+.main-nav ul li{
+  width:20%;
+  padding-top:30px;
+}
+blockquote{
+  text-align:center;
+  color:#fff;
+  font-style:italic;
+  position:relative;
+
+  margin-top:35px;
+}
+
+
+blockquote::before{
+  content: "";
+  background: url(image_path("wheat-left.png")) no-repeat;
+  display: inline-block;
+  min-height: 100px;
+  min-width: 110px;
+  position: absolute;
+  left: -95px;
+  top: -30px;
+}
+
+blockquote::after{
+  content: "";
+  background: url(image_path("wheat-right.png")) no-repeat;
+  display: inline-block;
+  min-height: 100px;
+  min-width: 110px;
+  position: absolute;
+  right: -95px;
+  top: -30px;
+}
+
+.logo{
+  margin:-62px 0 0 -25px;
+
+}
+.logo img{
+  width: 135px;
+  height: 188px;
+}
+
+/*BEERS*/
+
+/* FOOT */
+
+/* HOME */
+
+img{
+  width:100%;
+}
+
+body{
+  background:url(image_path("chalk.jpg")) repeat;
+  color:#fff;
+
+}
+.staticpages_hero{
+  background:url(image_path("brewery.jpg")) no-repeat ;
+  min-height:500px;
+  max-height:500px;
+  background-size:100%;
+}
+
+.about{
+  background:url(image_path("rust.jpg")) repeat ;
+  background-size:100%;
+
+}
+
+.quote {
+  background:url(image_path("chalk.jpg")) repeat ;
+
+}
+
+footer{
+  background:url(image_path("border.png")) repeat-x #000;
+  min-height:70px;
+}
+
+.contact{
+  background:url(image_path("beer-bottles.jpg")) no-repeat ;
+}
+
+/*BEERS*/
+
+ul.categories-nav li{
+  font-family:'Arvo', serif;
+  font-weight:bold;
+  letter-spacing:.006em;
+  color: #fff;
+  display: inline;
+  float: left;
+  font-size: 30px;
+  padding-right: 20px;
+  text-transform: uppercase;
+}
+
+.beer_bottle{
+  margin-top: -80px;
+  margin-left: -50px;
+}
+
+
+/* CART */
+
+
+.stitched{
+  border:2px dashed #fff;
+  border-radius: 5px;
+  padding:20px;
+}
+table{
+  width: 94%;
+  margin: 0px 3%;
+}
+
+th, tr{
+  width:20%;
+  text-align:center!important;
+}
+tr {
+   border-bottom: 2px dashed #fff;
+}
+.basket{
+  position:relative;
+}
+
+.basket .button, button, input[type="submit"], input[type="reset"], input[type="button"]{
+  background-color:#8a1f1a;
+  border:none;
+  font-family: 'Oswald', sans-serif;;
+  color:#fff;
+  float:left;
+}
+
+.basket{
+  padding: 50px 0px;
+}
+.basket p{
+  float:right;
+  position:relative;
+  padding-right:30px;
+}
+.basket p:before{
+  content: "";
+  background: url(image_path("basket.png")) no-repeat;
+  display: block;
+  min-height: 44px;
+  min-width: 50px;
+  position: absolute;
+  left: -61px;
+  top: -7px;
+}
+
+.body{
+  background-color:#000!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Colon after property should be followed by one space